usersService/src/main/java/il/ac/afeka/usersservice/boundaries/UserBoundary.java

```diff
@@ -10,22 +14,26 @@ public class UserBoundary {
     private String recruitdate;
     private String[] roles;
 
-    public UserBoundary() {}
+    public UserBoundary() {
+    }
 
     public UserBoundary(UserEntity user) {
         this.email = user.getEmail();
         this.name = new NameBoundary(user.getFirstname(), user.getLastname());
```

Comment:
need to check for name valid. (without numbers and such)
found this on old project. 
"public static boolean validateName(String name) {
        // Define the regular expression for a valid name
        String regex = "^[a-zA-Z]+(\\s[a-zA-Z]+)?$";

        // Compile the regular expression
        Pattern pattern = Pattern.compile(regex);

        // Create a Matcher object
        Matcher matcher = pattern.matcher(name);

        // Check if the name matches the pattern
        return matcher.matches();
    }"  
but u should probably put in Utils.